optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -58,9 +59,13 @@ def gaudi_llama_attention_forward(
         key_states = self.k_proj(hidden_states)
         value_states = self.v_proj(hidden_states)
 
+    tp_world_size = 1 if self.training else int(os.environ.get("WORLD_SIZE", 1))
```

Comment:
could we add a TODO here, since MS DS has fixed [here](https://github.com/microsoft/DeepSpeed/blob/341cefd2a406f578ee05b3a9d0686553926a7e61/deepspeed/module_inject/auto_tp.py#L383)), and I have checked the code of Habana internal DS, same fixed is applied([here](https://github.com/HabanaAI/DeepSpeed/blob/main/deepspeed/module_inject/replace_module.py#L440)) but seems is not  released in 1.11.